### PR TITLE
[C#] Update introduction.md

### PR DESCRIPTION
**Pull request by pvcarrera**
Copy Ruby introduction to the C# repo.

### CODE REVIEW

**Comment by EarthlingRich** [comment]
This is way to complicated for a user that is just starting to learn code. Are these text shown to the student? If so, I like the previous option more. Maybe with an addition that a string represents text.

A `string` in C# is a sequence of Unicode characters (letters, digits, punctuation, etc.) that is used to represent text.

**Comment by pvcarrera** [comment]
@EarthlingRich let me double check, maybe I got this wrong. @iHiD is this the text you asked me to copy over in https://github.com/exercism/v3/pull/738 ?

**Comment by iHiD** [comment]
> Is this the text you asked me to copy over in #738 ?

@pvcarrera I didn't mean to ask you to copy this back to C# although I'm not against it in principle. However, I think editing the C# (maybe to include the reference of an object if that's true) is a good thing. But I'll leave that for @ErikSchierboom and C# maintainers to work out :)

> This is to complicated for a student that is just starting to learn coding

As a reminder, we're not targeting people learning to code. We're targeting people who know how to code but are learning C#.

**Comment by pvcarrera** [comment]
Ok, sorry for the misunderstanding. Feel free to close this PR if it's not useful for you.

**Comment by ErikSchierboom** [comment]
> This is to complicated for a student that is just starting to learn coding.

@EarthlingRich A little clarification: we are explicitly not targeting students that are just starting to learn coding. Our target audience are students that are proficient in at least one language. That said, I do think the above suggestion does not fit C# that well. But I do think it has some nice parts. Let me think of a modification.

File: languages/csharp/exercises/concept/strings/.docs/introduction.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Introduction
 
-A `string` in C# is a sequence of Unicode characters (letters, digits, punctuation, etc.).
+A `string` in C# is an object that holds and manipulates an arbitrary sequence of bytes, typically representing characters. Strings are manipulated by calling the string's methods.
```
Comment (ErikSchierboom) [inline languages/csharp/exercises/concept/strings/.docs/introduction.md]:
```suggestion
A `string` in C# is an object that holds a sequence of Unicode characters (letters, digits, punctuation, etc.). Strings are manipulated by calling the string's methods. Once a string has been constructed, its value can never change. Any methods that appear to modify a string will actually return a new string.
```

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/strings/.docs/introduction.md]: @EarthlingRich What do you think of the above suggestion?

Comment (mikedamay) [reply languages/csharp/exercises/concept/strings/.docs/introduction.md]: I would think it is worth giving one or two examples - reminding the traveller from Javascript or elsewhere that double quotes are mandatory and how we do verbatim strings in our language and maybe slip in an escaped character and a unicode code or two to show them how it's done.

Comment (EarthlingRich) [reply languages/csharp/exercises/concept/strings/.docs/introduction.md]: That is a great description @ErikSchierboom. The suggestions of @mikedamay are out of scope for this PR. It's more about the exercise itself.

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/strings/.docs/introduction.md]: > I would think it is worth giving one or two examples - reminding the traveller from Javascript or elsewhere that double quotes are mandatory and how we do verbatim strings in our language and maybe slip in an escaped character and a unicode code or two to show them how it's done.

@mikedamay The `instructions.md` document's aim is to provide a minimal description. The idea is that the student will figure these things out by themselves, as long as we'll provide them with the minimal context (which is why the text mentions `string` as the type).

@EarthlingRich Do you perhaps have a suggestion?

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/strings/.docs/introduction.md]: @EarthlingRich I've updated

Comment (EarthlingRich) [reply languages/csharp/exercises/concept/strings/.docs/introduction.md]: Great! 👍

**Pull request merged**